optimum/intel/openvino/configuration.py

```diff
@@ -83,6 +84,7 @@ def __init__(
         compression: Union[List[Dict], Dict, None] = None,
         input_info: Optional[List] = None,
         save_onnx_model: bool = False,
+        weight_quantization_config: Optional[QuantizationConfigMixin] = None,
```

Comment:
@echarlaix, fyi, if I name it `quantization_config` I get an error during the serialization process due to the logic in transformers library that checks the availability of `quantization_config` field and fails if it is None.

Reply:
Could having only one argument make sense instead of having both `compression` and `weight_quantization_config` (we could create an instance of `WeightQuantizationConfig` if needed in the init directly). Also could you add a test to verify serialization is working ?

Reply:
Fixed